src/TodoList.js

```diff
@@ -1,20 +1,49 @@
 class TodoList {
-    constructor(){
-        this.idCount = 1
-    }
+  constructor() {
+    this.items = [];
+    this.idCount = 1;
+  }
 
   create(description) {
     const todo = {
       id: this.idCount,
       text: description,
       status: "incomplete",
     };
-    this.idCount += 1
-    console.log(todo)
-    return todo
+    this.idCount += 1;
+    this.items.push(todo);
+    return todo;
+  }
+
+  setToCompleted(id) {
+    for (let i = 0; i < this.items.length; i++) {
+      if (this.items[i].id === id) {
+        this.items[i].status = "completed";
+        return this.items[i];
+      }
+    }
   }
 
+  getCompleted() {
+    let completed = this.items.filter((item) => item.status === "completed");
+    return completed;
+  }
 
+  getIncompleted() {
+    let incompleted = this.items.filter((item) => item.status === "incomplete");
+    return incompleted;
+  }
+
+  //this one is not completed - i dont know how to add alternative test in case "ID does not exist"
```

Comment:
Even if not complete, it's good you say this so person reviewing can offer a suggestion & knows this.